debezium-core/src/main/java/io/debezium/relational/RelationalChangeRecordEmitter.java

```diff
@@ -25,6 +26,9 @@
  */
 public abstract class RelationalChangeRecordEmitter extends AbstractChangeRecordEmitter<TableSchema> {
 
+    public static final String PK_UPDATE_OLDKEY_FIELD = "oldkey";
```

Comment:
what do you think about the const name and the header field name?

I was also thinking about `pk-update-ref-oldkey` and  `pk-update-ref-oldkey` or similar, or namespacing it somehow with `debezium`?

(applies also to next line)

Reply:
No opinion on the const name. Regarding header field names, that's a good point. We use "__debezium." as a prefix in some other places, so might be a good idea here, too.